src/constants.js

```diff
@@ -24,5 +24,15 @@ module.exports = {
   TRACE_AGENT_REQUEST_HEADER: 'x-cloud-trace-agent-request',
 
   /** @const {number} bitmask to determine whether trace is enabled in trace options. */
-  TRACE_OPTIONS_TRACE_ENABLED: 1 << 0
+  TRACE_OPTIONS_TRACE_ENABLED: 1 << 0,
+
+  /** See: cloud.google.com/trace/api/reference/rest/v1/projects.traces for limits. */
+  /** Maximum size of a span name in bytes. */
```

Comment:
This comment is now wrong. `Size of a span name (in bytes) must be less than this limit`.